### ISSUE TITLE
Change Staking requirement low

### ISSUE THREAD

**Issue by KeeJef** [loki-project/loki#358]
In the current economic formula the staking requirements decreases to 10,000 LOKI over the first 4 years of Loki's operation, after this 4 year decline the for the next 10 years the Loki staking requirement will increase up to 15,000 LOKI.

The current economic scheme would be incompatible with #346 for this reason i propose a minor change to the scheme, instead of rising to 15,000 after 4 years over a 10 year period the Loki staking requirement should drop to 15,000 at its lowest point after 4 years and continue to be 15,000 in perpetuity. 

The idea behind the increased staking requirement after year 4 was to encourage a higher node count over the first 4 years and once Loki was fully established and had a wide community increase the staking requirement to increase the $ROI for node operators. Its unclear whether this change would have that effect in reality and considering its effect on a major UX change #346 i think the case is clear for removal and change to the more simple case of 15,000 being the new lowest staking requirement.

**Comment by HIRAESH** [comment]
Since it does not have a big influence on the number of nodes, I think that there is no problem with 15,000 as the minimum number of LSR.

On the other hand, recent declines in LSR are too sharp. Originally, the LSR was designed when the concept of the pool does not exist. As the pool was designed after the LSR was proposed, owners of the pool are sufficient 25% owned Loki originally assumed.
Therefore, in order to obtain initially assumed Sybil attack resistance, it is necessary to make LSR decrease more slowly.

**Comment by Sonofotis** [comment]
Proposal:
At blockheight `235987` (scheduled hardfork date) add new staking requirement formula.

Keep current halflife from formula (loki_exp2) and change base and variable to have a clean LSR change at blockheight in which hardfork will occur.

General code should look like the below:
```
    uint64_t height_adjusted = height - hardfork_height;
    uint64_t base = 15000 * COIN;
    uint64_t variable = (24721.0 * COIN) / loki_exp2(height_adjusted/129600.0);
    return std::base + variable;
```

Wolfram:
Current: 
`=10000+(35000/(2^((h-101250)/129600)))`

Proposed:
`=15000+(24721/(2^((h-101250)/129600)))`

Need to do additional work on what is going to happen to the ROI. From what I can see it should have a positive effect to the ROI.

![lsr change](https://user-images.githubusercontent.com/39609800/49488702-df9c9380-f89b-11e8-822b-911fcf853e71.png)

<details>
<summary>Want to see the Data?</summary>
<br>

| Month       | BH      | Current LSR | Proposed LSR |
|-------------|---------|-------------|--------------|
| 0           | 101250  | 45000       | 45000        |
| 1           | 122850  | 41181.45513 | 41181.45513  |
| 2           | 144450  | 37779.51841 | 37779.51841  |
| 3           | 166050  | 34748.73734 | 34748.73734  |
| 4           | 187650  | 32048.61837 | 32048.61837  |
| 5           | 209250  | 29643.08585 | 29643.08585  |
| 6           | 230850  | 27500       | 27500        |
| 6.237824074 | 235987  | 27025.74085 | 27025.52399  |
| 7           | 252450  | 25590.72757 | 26011.95361  |
| 8           | 274050  | 23889.7592  | 24810.53535  |
| 9           | 295650  | 22374.36867 | 23740.19337  |
| 10          | 317250  | 21024.30919 | 22786.62707  |
| 11          | 338850  | 19821.54292 | 21937.09607  |
| 12          | 360450  | 18750       | 21180.25     |
| 13          | 382050  | 17795.36378 | 20505.9768   |
| 14          | 403650  | 16944.8796  | 19905.26768  |
| 15          | 425250  | 16187.18434 | 19370.09668  |
| 16          | 446850  | 15512.15459 | 18893.31353  |
| 17          | 468450  | 14910.77146 | 18468.54804  |
| 18          | 490050  | 14375       | 18090.125    |
| 19          | 511650  | 13897.68189 | 17752.9884   |
| 20          | 533250  | 13472.4398  | 17452.63384  |
| 21          | 554850  | 13093.59217 | 17185.04834  |
| 22          | 576450  | 12756.0773  | 16946.65677  |
| 23          | 598050  | 12455.38573 | 16734.27402  |
| 24          | 619650  | 12187.5     | 16545.0625   |
| 25          | 641250  | 11948.84095 | 16376.4942   |
| 26          | 662850  | 11736.2199  | 16226.31692  |
| 27          | 684450  | 11546.79608 | 16092.52417  |
| 28          | 706050  | 11378.03865 | 15973.32838  |
| 29          | 727650  | 11227.69287 | 15867.13701  |
| 30          | 749250  | 11093.75    | 15772.53125  |
| 31          | 770850  | 10974.42047 | 15688.2471   |
| 32          | 792450  | 10868.10995 | 15613.15846  |
| 33          | 814050  | 10773.39804 | 15546.26209  |
| 34          | 835650  | 10689.01932 | 15486.66419  |
| 35          | 857250  | 10613.84643 | 15433.5685   |
| 36          | 878850  | 10546.875   | 15386.26563  |
| 37          | 900450  | 10487.21024 | 15344.12355  |
| 38          | 922050  | 10434.05498 | 15306.57923  |
| 39          | 943650  | 10386.69902 | 15273.13104  |
| 40          | 965250  | 10344.50966 | 15243.3321   |
| 41          | 986850  | 10306.92322 | 15216.78425  |
| 42          | 1008450 | 10273.4375  | 15193.13281  |
| 43          | 1030050 | 10243.60512 | 15172.06178  |
| 44          | 1051650 | 10217.02749 | 15153.28961  |
| 45          | 1073250 | 10193.34951 | 15136.56552  |
| 46          | 1094850 | 10172.25483 | 15121.66605  |
| 47          | 1116450 | 10153.64583 | 15108.39213  |
| 48          | 1138050 | 10195.3125  | 15096.56641  |
| 49          | 1159650 | 10236.97917 | 15086.03089  |
| 50          | 1181250 | 10278.64583 | 15076.64481  |
| 51          | 1202850 | 10320.3125  | 15068.28276  |
| 52          | 1224450 | 10361.97917 | 15060.83302  |
| 53          | 1246050 | 10403.64583 | 15054.19606  |
| 54          | 1267650 | 10445.3125  | 15048.2832   |
| 55          | 1289250 | 10486.97917 | 15043.01544  |
| 56          | 1310850 | 10528.64583 | 15038.3224   |
| 57          | 1332450 | 10570.3125  | 15034.14138  |
| 58          | 1354050 | 10611.97917 | 15030.41651  |
| 59          | 1375650 | 10653.64583 | 15027.09803  |
| 60          | 1397250 | 10695.3125  | 15024.1416   |
| 61          | 1418850 | 10736.97917 | 15021.50772  |
| 62          | 1440450 | 10778.64583 | 15019.1612   |
| 63          | 1462050 | 10820.3125  | 15017.07069  |
| 64          | 1483650 | 10861.97917 | 15015.20826  |
| 65          | 1505250 | 10903.64583 | 15013.54902  |
| 66          | 1526850 | 10945.3125  | 15012.0708   |
| 67          | 1548450 | 10986.97917 | 15010.75386  |
| 68          | 1570050 | 11028.64583 | 15009.5806   |
| 69          | 1591650 | 11070.3125  | 15008.53535  |
| 70          | 1613250 | 11111.97917 | 15007.60413  |
| 71          | 1634850 | 11153.64583 | 15006.77451  |
| 72          | 1656450 | 11195.3125  | 15006.0354   |
| 73          | 1678050 | 11236.97917 | 15005.37693  |
| 74          | 1699650 | 11278.64583 | 15004.7903   |
| 75          | 1721250 | 11320.3125  | 15004.26767  |
| 76          | 1742850 | 11361.97917 | 15003.80206  |
| 77          | 1764450 | 11403.64583 | 15003.38725  |
| 78          | 1786050 | 11445.3125  | 15003.0177   |
| 79          | 1807650 | 11486.97917 | 15002.68847  |
| 80          | 1829250 | 11528.64583 | 15002.39515  |
| 81          | 1850850 | 11570.3125  | 15002.13384  |
| 82          | 1872450 | 11611.97917 | 15001.90103  |
| 83          | 1894050 | 11653.64583 | 15001.69363  |
| 84          | 1915650 | 11695.3125  | 15001.50885  |
| 85          | 1937250 | 11736.97917 | 15001.34423  |
| 86          | 1958850 | 11778.64583 | 15001.19758  |
| 87          | 1980450 | 11820.3125  | 15001.06692  |
| 88          | 2002050 | 11861.97917 | 15000.95052  |
| 89          | 2023650 | 11903.64583 | 15000.84681  |
| 90          | 2045250 | 11945.3125  | 15000.75443  |
| 91          | 2066850 | 11986.97917 | 15000.67212  |
| 92          | 2088450 | 12028.64583 | 15000.59879  |
| 93          | 2110050 | 12070.3125  | 15000.53346  |
| 94          | 2131650 | 12111.97917 | 15000.47526  |
| 95          | 2153250 | 12153.64583 | 15000.42341  |
| 96          | 2174850 | 12195.3125  | 15000.37721  |
| 97          | 2196450 | 12236.97917 | 15000.33606  |
| 98          | 2218050 | 12278.64583 | 15000.29939  |
| 99          | 2239650 | 12320.3125  | 15000.26673  |
| 100         | 2261250 | 12361.97917 | 15000.23763  |
| 101         | 2282850 | 12403.64583 | 15000.2117   |
| 102         | 2304450 | 12445.3125  | 15000.18861  |
| 103         | 2326050 | 12486.97917 | 15000.16803  |
| 104         | 2347650 | 12528.64583 | 15000.1497   |
| 105         | 2369250 | 12570.3125  | 15000.13336  |
| 106         | 2390850 | 12611.97917 | 15000.11881  |
| 107         | 2412450 | 12653.64583 | 15000.10585  |
| 108         | 2434050 | 12695.3125  | 15000.0943   |
| 109         | 2455650 | 12736.97917 | 15000.08401  |
| 110         | 2477250 | 12778.64583 | 15000.07485  |
| 111         | 2498850 | 12820.3125  | 15000.06668  |
| 112         | 2520450 | 12861.97917 | 15000.05941  |
| 113         | 2542050 | 12903.64583 | 15000.05293  |
| 114         | 2563650 | 12945.3125  | 15000.04715  |
| 115         | 2585250 | 12986.97917 | 15000.04201  |
| 116         | 2606850 | 13028.64583 | 15000.03742  |
| 117         | 2628450 | 13070.3125  | 15000.03334  |
| 118         | 2650050 | 13111.97917 | 15000.0297   |
| 119         | 2671650 | 13153.64583 | 15000.02646  |
| 120         | 2693250 | 13195.3125  | 15000.02358  |
| 121         | 2714850 | 13236.97917 | 15000.021    |
| 122         | 2736450 | 13278.64583 | 15000.01871  |
| 123         | 2758050 | 13320.3125  | 15000.01667  |
| 124         | 2779650 | 13361.97917 | 15000.01485  |
| 125         | 2801250 | 13403.64583 | 15000.01323  |
| 126         | 2822850 | 13445.3125  | 15000.01179  |
| 127         | 2844450 | 13486.97917 | 15000.0105   |
| 128         | 2866050 | 13528.64583 | 15000.00936  |
| 129         | 2887650 | 13570.3125  | 15000.00834  |
| 130         | 2909250 | 13611.97917 | 15000.00743  |
| 131         | 2930850 | 13653.64583 | 15000.00662  |
| 132         | 2952450 | 13695.3125  | 15000.00589  |
| 133         | 2974050 | 13736.97917 | 15000.00525  |
| 134         | 2995650 | 13778.64583 | 15000.00468  |
| 135         | 3017250 | 13820.3125  | 15000.00417  |
| 136         | 3038850 | 13861.97917 | 15000.00371  |
| 137         | 3060450 | 13903.64583 | 15000.00331  |
| 138         | 3082050 | 13945.3125  | 15000.00295  |
| 139         | 3103650 | 13986.97917 | 15000.00263  |
| 140         | 3125250 | 14028.64583 | 15000.00234  |
| 141         | 3146850 | 14070.3125  | 15000.00208  |
| 142         | 3168450 | 14111.97917 | 15000.00186  |
| 143         | 3190050 | 14153.64583 | 15000.00165  |
| 144         | 3211650 | 14195.3125  | 15000.00147  |
| 145         | 3233250 | 14236.97917 | 15000.00131  |
| 146         | 3254850 | 14278.64583 | 15000.00117  |
| 147         | 3276450 | 14320.3125  | 15000.00104  |
| 148         | 3298050 | 14361.97917 | 15000.00093  |
| 149         | 3319650 | 14403.64583 | 15000.00083  |
| 150         | 3341250 | 14445.3125  | 15000.00074  |
| 151         | 3362850 | 14486.97917 | 15000.00066  |
| 152         | 3384450 | 14528.64583 | 15000.00058  |
| 153         | 3406050 | 14570.3125  | 15000.00052  |
| 154         | 3427650 | 14611.97917 | 15000.00046  |
| 155         | 3449250 | 14653.64583 | 15000.00041  |
| 156         | 3470850 | 14695.3125  | 15000.00037  |
| 157         | 3492450 | 14736.97917 | 15000.00033  |
| 158         | 3514050 | 14778.64583 | 15000.00029  |
| 159         | 3535650 | 14820.3125  | 15000.00026  |
| 160         | 3557250 | 14861.97917 | 15000.00023  |
| 161         | 3578850 | 14903.64583 | 15000.00021  |
| 162         | 3600450 | 14945.3125  | 15000.00018  |
| 163         | 3622050 | 14986.97917 | 15000.00016  |
| 164         | 3643650 | 15000       | 15000.00015  |
<details>

**Comment by Sonofotis** [comment]
Proposal:
At blockheight `~235987` add new staking requirement formula.

Keep current halflife from formula (loki_exp2) and change base and variable to have a clean LSR change at blockheight in which hardfork will occur.

General code should look like the below:
```
    uint64_t height_adjusted = height - hardfork_height;
    uint64_t base = 15000 * COIN;
    uint64_t variable = (24721.0 * COIN) / loki_exp2(height_adjusted/129600.0);
    return std::base + variable;
```

Wolfram:
Current: 
`=10000+(35000/(2^((h-101250)/129600)))`

Proposed:
`=15000+(24721/(2^((h-101250)/129600)))`

Need to do additional work on what is going to happen to the ROI. From what I can see it should have a positive effect to the ROI.

![lsr change](https://user-images.githubusercontent.com/39609800/49488702-df9c9380-f89b-11e8-822b-911fcf853e71.png)

<details>
<summary>Want to see the Data?</summary>
<br>

| Month       | BH      | Current LSR | Proposed LSR |
|-------------|---------|-------------|--------------|
| 0           | 101250  | 45000       | 45000        |
| 1           | 122850  | 41181.45513 | 41181.45513  |
| 2           | 144450  | 37779.51841 | 37779.51841  |
| 3           | 166050  | 34748.73734 | 34748.73734  |
| 4           | 187650  | 32048.61837 | 32048.61837  |
| 5           | 209250  | 29643.08585 | 29643.08585  |
| 6           | 230850  | 27500       | 27500        |
| 6.237824074 | 235987  | 27025.74085 | 27025.52399  |
| 7           | 252450  | 25590.72757 | 26011.95361  |
| 8           | 274050  | 23889.7592  | 24810.53535  |
| 9           | 295650  | 22374.36867 | 23740.19337  |
| 10          | 317250  | 21024.30919 | 22786.62707  |
| 11          | 338850  | 19821.54292 | 21937.09607  |
| 12          | 360450  | 18750       | 21180.25     |
| 13          | 382050  | 17795.36378 | 20505.9768   |
| 14          | 403650  | 16944.8796  | 19905.26768  |
| 15          | 425250  | 16187.18434 | 19370.09668  |
| 16          | 446850  | 15512.15459 | 18893.31353  |
| 17          | 468450  | 14910.77146 | 18468.54804  |
| 18          | 490050  | 14375       | 18090.125    |
| 19          | 511650  | 13897.68189 | 17752.9884   |
| 20          | 533250  | 13472.4398  | 17452.63384  |
| 21          | 554850  | 13093.59217 | 17185.04834  |
| 22          | 576450  | 12756.0773  | 16946.65677  |
| 23          | 598050  | 12455.38573 | 16734.27402  |
| 24          | 619650  | 12187.5     | 16545.0625   |
| 25          | 641250  | 11948.84095 | 16376.4942   |
| 26          | 662850  | 11736.2199  | 16226.31692  |
| 27          | 684450  | 11546.79608 | 16092.52417  |
| 28          | 706050  | 11378.03865 | 15973.32838  |
| 29          | 727650  | 11227.69287 | 15867.13701  |
| 30          | 749250  | 11093.75    | 15772.53125  |
| 31          | 770850  | 10974.42047 | 15688.2471   |
| 32          | 792450  | 10868.10995 | 15613.15846  |
| 33          | 814050  | 10773.39804 | 15546.26209  |
| 34          | 835650  | 10689.01932 | 15486.66419  |
| 35          | 857250  | 10613.84643 | 15433.5685   |
| 36          | 878850  | 10546.875   | 15386.26563  |
| 37          | 900450  | 10487.21024 | 15344.12355  |
| 38          | 922050  | 10434.05498 | 15306.57923  |
| 39          | 943650  | 10386.69902 | 15273.13104  |
| 40          | 965250  | 10344.50966 | 15243.3321   |
| 41          | 986850  | 10306.92322 | 15216.78425  |
| 42          | 1008450 | 10273.4375  | 15193.13281  |
| 43          | 1030050 | 10243.60512 | 15172.06178  |
| 44          | 1051650 | 10217.02749 | 15153.28961  |
| 45          | 1073250 | 10193.34951 | 15136.56552  |
| 46          | 1094850 | 10172.25483 | 15121.66605  |
| 47          | 1116450 | 10153.64583 | 15108.39213  |
| 48          | 1138050 | 10195.3125  | 15096.56641  |
| 49          | 1159650 | 10236.97917 | 15086.03089  |
| 50          | 1181250 | 10278.64583 | 15076.64481  |
| 51          | 1202850 | 10320.3125  | 15068.28276  |
| 52          | 1224450 | 10361.97917 | 15060.83302  |
| 53          | 1246050 | 10403.64583 | 15054.19606  |
| 54          | 1267650 | 10445.3125  | 15048.2832   |
| 55          | 1289250 | 10486.97917 | 15043.01544  |
| 56          | 1310850 | 10528.64583 | 15038.3224   |
| 57          | 1332450 | 10570.3125  | 15034.14138  |
| 58          | 1354050 | 10611.97917 | 15030.41651  |
| 59          | 1375650 | 10653.64583 | 15027.09803  |
| 60          | 1397250 | 10695.3125  | 15024.1416   |
| 61          | 1418850 | 10736.97917 | 15021.50772  |
| 62          | 1440450 | 10778.64583 | 15019.1612   |
| 63          | 1462050 | 10820.3125  | 15017.07069  |
| 64          | 1483650 | 10861.97917 | 15015.20826  |
| 65          | 1505250 | 10903.64583 | 15013.54902  |
| 66          | 1526850 | 10945.3125  | 15012.0708   |
| 67          | 1548450 | 10986.97917 | 15010.75386  |
| 68          | 1570050 | 11028.64583 | 15009.5806   |
| 69          | 1591650 | 11070.3125  | 15008.53535  |
| 70          | 1613250 | 11111.97917 | 15007.60413  |
| 71          | 1634850 | 11153.64583 | 15006.77451  |
| 72          | 1656450 | 11195.3125  | 15006.0354   |
| 73          | 1678050 | 11236.97917 | 15005.37693  |
| 74          | 1699650 | 11278.64583 | 15004.7903   |
| 75          | 1721250 | 11320.3125  | 15004.26767  |
| 76          | 1742850 | 11361.97917 | 15003.80206  |
| 77          | 1764450 | 11403.64583 | 15003.38725  |
| 78          | 1786050 | 11445.3125  | 15003.0177   |
| 79          | 1807650 | 11486.97917 | 15002.68847  |
| 80          | 1829250 | 11528.64583 | 15002.39515  |
| 81          | 1850850 | 11570.3125  | 15002.13384  |
| 82          | 1872450 | 11611.97917 | 15001.90103  |
| 83          | 1894050 | 11653.64583 | 15001.69363  |
| 84          | 1915650 | 11695.3125  | 15001.50885  |
| 85          | 1937250 | 11736.97917 | 15001.34423  |
| 86          | 1958850 | 11778.64583 | 15001.19758  |
| 87          | 1980450 | 11820.3125  | 15001.06692  |
| 88          | 2002050 | 11861.97917 | 15000.95052  |
| 89          | 2023650 | 11903.64583 | 15000.84681  |
| 90          | 2045250 | 11945.3125  | 15000.75443  |
| 91          | 2066850 | 11986.97917 | 15000.67212  |
| 92          | 2088450 | 12028.64583 | 15000.59879  |
| 93          | 2110050 | 12070.3125  | 15000.53346  |
| 94          | 2131650 | 12111.97917 | 15000.47526  |
| 95          | 2153250 | 12153.64583 | 15000.42341  |
| 96          | 2174850 | 12195.3125  | 15000.37721  |
| 97          | 2196450 | 12236.97917 | 15000.33606  |
| 98          | 2218050 | 12278.64583 | 15000.29939  |
| 99          | 2239650 | 12320.3125  | 15000.26673  |
| 100         | 2261250 | 12361.97917 | 15000.23763  |
| 101         | 2282850 | 12403.64583 | 15000.2117   |
| 102         | 2304450 | 12445.3125  | 15000.18861  |
| 103         | 2326050 | 12486.97917 | 15000.16803  |
| 104         | 2347650 | 12528.64583 | 15000.1497   |
| 105         | 2369250 | 12570.3125  | 15000.13336  |
| 106         | 2390850 | 12611.97917 | 15000.11881  |
| 107         | 2412450 | 12653.64583 | 15000.10585  |
| 108         | 2434050 | 12695.3125  | 15000.0943   |
| 109         | 2455650 | 12736.97917 | 15000.08401  |
| 110         | 2477250 | 12778.64583 | 15000.07485  |
| 111         | 2498850 | 12820.3125  | 15000.06668  |
| 112         | 2520450 | 12861.97917 | 15000.05941  |
| 113         | 2542050 | 12903.64583 | 15000.05293  |
| 114         | 2563650 | 12945.3125  | 15000.04715  |
| 115         | 2585250 | 12986.97917 | 15000.04201  |
| 116         | 2606850 | 13028.64583 | 15000.03742  |
| 117         | 2628450 | 13070.3125  | 15000.03334  |
| 118         | 2650050 | 13111.97917 | 15000.0297   |
| 119         | 2671650 | 13153.64583 | 15000.02646  |
| 120         | 2693250 | 13195.3125  | 15000.02358  |
| 121         | 2714850 | 13236.97917 | 15000.021    |
| 122         | 2736450 | 13278.64583 | 15000.01871  |
| 123         | 2758050 | 13320.3125  | 15000.01667  |
| 124         | 2779650 | 13361.97917 | 15000.01485  |
| 125         | 2801250 | 13403.64583 | 15000.01323  |
| 126         | 2822850 | 13445.3125  | 15000.01179  |
| 127         | 2844450 | 13486.97917 | 15000.0105   |
| 128         | 2866050 | 13528.64583 | 15000.00936  |
| 129         | 2887650 | 13570.3125  | 15000.00834  |
| 130         | 2909250 | 13611.97917 | 15000.00743  |
| 131         | 2930850 | 13653.64583 | 15000.00662  |
| 132         | 2952450 | 13695.3125  | 15000.00589  |
| 133         | 2974050 | 13736.97917 | 15000.00525  |
| 134         | 2995650 | 13778.64583 | 15000.00468  |
| 135         | 3017250 | 13820.3125  | 15000.00417  |
| 136         | 3038850 | 13861.97917 | 15000.00371  |
| 137         | 3060450 | 13903.64583 | 15000.00331  |
| 138         | 3082050 | 13945.3125  | 15000.00295  |
| 139         | 3103650 | 13986.97917 | 15000.00263  |
| 140         | 3125250 | 14028.64583 | 15000.00234  |
| 141         | 3146850 | 14070.3125  | 15000.00208  |
| 142         | 3168450 | 14111.97917 | 15000.00186  |
| 143         | 3190050 | 14153.64583 | 15000.00165  |
| 144         | 3211650 | 14195.3125  | 15000.00147  |
| 145         | 3233250 | 14236.97917 | 15000.00131  |
| 146         | 3254850 | 14278.64583 | 15000.00117  |
| 147         | 3276450 | 14320.3125  | 15000.00104  |
| 148         | 3298050 | 14361.97917 | 15000.00093  |
| 149         | 3319650 | 14403.64583 | 15000.00083  |
| 150         | 3341250 | 14445.3125  | 15000.00074  |
| 151         | 3362850 | 14486.97917 | 15000.00066  |
| 152         | 3384450 | 14528.64583 | 15000.00058  |
| 153         | 3406050 | 14570.3125  | 15000.00052  |
| 154         | 3427650 | 14611.97917 | 15000.00046  |
| 155         | 3449250 | 14653.64583 | 15000.00041  |
| 156         | 3470850 | 14695.3125  | 15000.00037  |
| 157         | 3492450 | 14736.97917 | 15000.00033  |
| 158         | 3514050 | 14778.64583 | 15000.00029  |
| 159         | 3535650 | 14820.3125  | 15000.00026  |
| 160         | 3557250 | 14861.97917 | 15000.00023  |
| 161         | 3578850 | 14903.64583 | 15000.00021  |
| 162         | 3600450 | 14945.3125  | 15000.00018  |
| 163         | 3622050 | 14986.97917 | 15000.00016  |
| 164         | 3643650 | 15000       | 15000.00015  |
<details>

**Comment by HIRAESH** [comment]
Since the decrease in LSR is steep, how about linearizing the reduction rate?

For example,
At blockheight `~230851` add new staking requirement formula.

Proposed_2:
`=MAX(15000,27500-(0.00362*(h-230850)))`

![linear_1](https://user-images.githubusercontent.com/45561460/50380714-e7d63a80-06b3-11e9-9298-d1d55e6c37e6.jpg)

**Comment by KeeJef** [comment]
Closed as of #422
